### PR TITLE
Added currentDirectory method "curdir"

### PR DESCRIPTION
**Pull request by BPL92022**
curdir will show the current path

### CODE REVIEW

File: src/main/scala/FileSystem.scala [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    val inputKeyboard = getKeyboardInput
+
+    val cmdMatcher = inputKeyboard match{
+      case "q" => "The key q is pressed, the app will close."
+      case "h" =>  help(); commandInterpretor()
+      case "curdir" => println( getCurrentPath ); commandInterpretor()
+      case _   => println("The command is not recognized. Type \"h\" to list the commands and how to use them or \"q\" to quit."); commandInterpretor()
+    }
+
+    println( cmdMatcher )
```
Comment (igalatanu) [inline src/main/scala/FileSystem.scala:15]: Do you need this? I believe the type of cmdMatcher is/should be `Unit`, so all this method should contain only the first print and the pattern matching

File: src/main/scala/FileSystem.scala [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+class FileSystem {
+  def commandInterpretor() : Unit = {
+    print( "Your cmd (press \"q\" to quit, \"h\" for help): " )
+    val inputKeyboard = getKeyboardInput
+
+    val cmdMatcher = inputKeyboard match{
+      case "q" => "The key q is pressed, the app will close."
+      case "h" =>  help(); commandInterpretor()
+      case "curdir" => println( getCurrentPath ); commandInterpretor()
+      case _   => println("The command is not recognized. Type \"h\" to list the commands and how to use them or \"q\" to quit."); commandInterpretor()
```
Comment (igalatanu) [inline src/main/scala/FileSystem.scala]: In Scala, you can use triple quotes so you don't need to escape your inner quotes
```suggestion
      case _   => println("""The command is not recognized. Type "h" to list the commands and how to use them or "q" to quit."""); commandInterpretor()
```

File: src/main/scala/FileSystem.scala [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    }
+
+    println( cmdMatcher )
+  }
+
+  def help() : Unit = {
+    println(
+      "Run the following commands (without quotes):\n" +
+        "\"curdir\" = will show the current directory"
```
Comment (igalatanu) [inline src/main/scala/FileSystem.scala]: Same here. You can use triple quotes for better and clearer formatting
```suggestion
      """Run the following commands (without quotes):
        |"curdir" = will show the current directory
        |""".stripMargin
```

**Review by igalatanu — COMMENTED** [review]
Nice start! I like that you created an interpretor for your commands, which is more functional-like.
The only remark i would have is that the idea of the exercise is to model the filesystem and it's commands, so it all should be a simulation which doesn't use the SO filesystem. Using `java.io.File` is more of a cheat. I recommend creating models for Files/Directories and also for the commands you used.

File: src/main/scala/FileSystem.scala (hunk 3 above)
Comment (BPL92022) [reply src/main/scala/FileSystem.scala]: > Nice start! I like that you created an interpretor for your commands, which is more functional-like. The only remark i would have is that the idea of the exercise is to model the filesystem and it's commands, so it all should be a simulation which doesn't use the SO filesystem. Using `java.io.File` is more of a cheat. I recommend creating models for Files/Directories and also for the commands you used.

Thank you, I will re-think and re-format to be in Simulation-like and not real file-handling.

**Pull request closed without merging**